### PR TITLE
Config codecov.yml to prevent codecov status check blocking pull request.

### PR DESCRIPTION
**Pull request by yangchiu**
Config codecov.yml to prevent codecov status check blocking pull request.

For https://github.com/longhorn/longhorn/issues/4425

Signed-off-by: Yang Chiu <yang.chiu@suse.com>

### CODE REVIEW

File: codecov.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+        base: auto
+        flags:
+          - unit
+        paths:
+          - "src"
+       # advanced settings
+        branches:
+          - master
+        if_ci_failed: error #success, failure, error, ignore
+        informational: false
```
Comment (innobead) [inline codecov.yml:17]: Should information be `true`?

Comment (yangchiu) [reply codecov.yml]: Sorry for the confusing. The old commit is just default values. I've updated it and it should disable both project and patch status check.

Comment (innobead) [reply codecov.yml]: Thanks 👍 

Two things here for PR.
- Can we still see the coverage in each PR status check? but don't mark the checks failed even if the coverage decreases. (I feel the informational setting of the patch status can help)
- Given the first, don't add a coverage comment in the PR

Comment (yangchiu) [reply codecov.yml]: I've modified it and it should disable comment and let PR status check always pass.

**Review by innobead — APPROVED** [review]

**Pull request merged**